shell.nix

```diff
@@ -5,6 +5,6 @@ in
   haskell.lib.buildStackProject {
      ghc = haskell.packages.${lts}.ghc;
      name = "rscoin";
-     buildInputs = [ zlib glib cairo gnome.pango gnome3.gtk git cabal-install
+     buildInputs = [ zlib.out zlib.dev glib cairo gnome.pango gnome3.gtk git cabal-install
```

Comment:
I did manage to get it to work, but I didn't want to delete nix-shell file. But we can go that route if it's preferred.